config/default.yml

```
@@ -119,6 +119,15 @@ Rails/BulkChangeTable:
   Include:
     - db/migrate/*.rb
 
+Rails/ContentTag:
+  Description: 'Use `tag` instead of `content_tag`.'
+  Reference:
+    - 'https://github.com/rails/rails/issues/25195'
+    - 'https://api.rubyonrails.org/classes/ActionView/Helpers/TagHelper.html#method-i-content_tag'
+  Enabled: false
+  Safe: true
+  VersionAdded: '2.5'
```

RuboCop Rails 2.5 series has been released. I think that it can be added in RuboCop Rails 2.6.

```suggestion
  VersionAdded: '2.6'
```